src/current/_includes/cockroachcloud/first-org-user-roles.md

```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_info}}
```

Comment:
Suggestion: what about linking the roles to the descriptions on the Authorization page, because you have those nice section headers for each to make it v easy

Reply:
This may be a silly question: is the first user in a newly created organization always the person who has created that organization? Or, can the first user be added?

Reply:
I updated the wording to say the user who creates the org, i think that is clearer (and answers your questions!).
My inclination is not to do the links because I hate links inside includes that will be included from both cloud and self hosted, it's a huge pain. There are other links to those roles on the page. So I'll skip it unless you have a strong opinion?